helm/prometheus-rules/templates/platform/atlas/recording-rules/grafana-cloud.rules.yml

```diff
@@ -574,3 +574,16 @@ spec:
             "(.*)"
           )
         record: aggregation:slo:time_period:days
+      - expr: |-
+          (
+            count(up{}== 0 )
```

Comment:
Why do we need the division? is the first one not enough?

Reply:
Do you mind adding a comment here though?

Also, maybe we want to use this recording rule in the alert instead?

Reply:
Comments added :heavy_check_mark: 
We can probably use the recording rule in the alert as well, but I'd like to play with the generated metrics in a larger scale before.